.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'ExpediaGroup/dropwizard-template-config'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Java 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          server-id: ossrh
+          server-username: SONATYPE_USERNAME # env variable for username in deploy
```

Comment:
shouldn't this be `${ { secrets.SONATYPE_USERNAME }}`?

Reply:
actually i dont think you need those env variables to set up java -> they are only needed for maven job

Reply:
Setting these variables will also setup the `settings.xml` file for Maven publishing in one spot (setup Java and Maven).
https://github.com/actions/setup-java#publishing-using-apache-maven

Those variables will produce a `settings.xml` file like this:
```
<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
  <servers>
    <server>
      <id>maven</id>
      <username>${env.SONATYPE_USERNAME}</username>
      <password>${env.SONATYPE_PASSWORD}</password>
    </server>
    <server>
      <id>gpg.passphrase</id>
      <passphrase>${env.GPG_PASSPHRASE}</passphrase>
    </server>
  </servers>
</settings>
```
And those environment variables are later set during the publish step.

Reply:
nice!